api/src/main/java/com/netflix/iceberg/expressions/InclusiveMetricsEvaluator.java

```diff
@@ -53,12 +54,20 @@ private MetricsEvalVisitor visitor() {
     return visitors.get();
   }
 
-  public InclusiveMetricsEvaluator(Schema schema, Expression unbound) {
+  InclusiveMetricsEvaluator(Schema schema, Expression unbound) {
```

Comment:
I see you've also changed the access modifier for this particular constructor, now package-private, guessing you're planning to retire this constructor since it's no longer public? It may still be accessible by subclasses of `InclusiveMetricsEvaluator` only within the same package though, right?

Reply:
Right, we did similar changes in #82 so that there are minimal changes in current tests, yet forcing most callers to make a choice on case sensitivity.